Sources/GoogleMapsUtils/Clustering/Algorithms/GridBasedCluster/GMUGridBasedClusterAlgorithm1.swift

```diff
@@ -58,19 +58,22 @@ final class GMUGridBasedClusterAlgorithm1: GMUClusterAlgorithm1 {
         var clusters: [Int : GMUCluster1] = [:]
 
         // Divide the whole map into a numCells x numCells grid and assign items to them.
-        let numCells: Double = Double(ceil(256 * pow(2, zoom) / Float(gmuGridCellSizePoints)))
+        let numCells: Int = Int(ceil(256 * pow(2, zoom) / gmuGridCellSizePoints))
 
         for item in clusterItems {
             let point: GMSMapPoint = GMSProject(item.position)
             /// point.x is in [-1, 1] range
-            let col: Double = numCells * (1.0 + point.x) / 2
+            let col: Int = Int(Double(numCells) * (1.0 + point.x) / 2.0)
             /// point.y is in [-1, 1] range
-            let row: Double = numCells * (1.0 + point.y) / 2
-            let index: Int = Int(numCells * row + col)
+            let row: Int = Int(Double(numCells) * (1.0 + point.y) / 2.0)
+            let index: Int = numCells * row + col
             var cluster: GMUStaticCluster1? = clusters[index] as? GMUStaticCluster1
             if cluster == nil {
                 // Normalize cluster's centroid to center of the cell.
-                let mapPoint: GMSMapPoint = GMSMapPoint(x: (col + 0.5) * 2.0 / numCells - 1, y: (row + 0.5) * 2.0 / numCells - 1)
```

Comment:
[self-review]: Broken down the complex expression to multiple simple expression, to speed up the xcode build process.